doc-include/migrate-mac.dtx

```diff
@@ -358,14 +358,14 @@
 %   \item \cs{unvxh} has been replaced with \cs{Xunvxh} and \cs{unvxhX} with two mandatory arguments.
 % \end{itemize}
 % \subsection{Migration to \reledmac~2.1.0}
-% \Reledmac~2.1.0 fix some bug when using \cs{Xbhooknote} and \cs{bhooknoteX} not in order to execute code at the beginning of each notes, but to insert content of at the beginning of each notes.
+% \Reledmac~2.1.0 fix some bugs when using \cs{Xbhooknote} and \cs{bhooknoteX}, in order to insert content of at the beginning of each note. This is not intended to execute code at the beginning of each note.
```

Comment:
@bishopcranmer why do you change the position. You say exactly the opposition as I say. I would like to say people should not use bhooknote to insert content, but should use it to execute code...